volume/helpers_test.go

```diff
@@ -29,27 +29,27 @@ const fixture = `
 `
 
 func TestStatParse(t *testing.T) {
-	attr, path, err := StatParse("96 1550611510 1550611448 1550611448 41ed mnt/path")
+	actualAttr, path, err := StatParse("96 1550611510 1550611448 1550611448 41ed mnt/path")
 	assert.Nil(t, err)
 	assert.Equal(t, "mnt/path", path)
-	assert.Equal(t, plugin.Attributes{
-		Atime: time.Unix(1550611510, 0),
-		Mtime: time.Unix(1550611448, 0),
-		Ctime: time.Unix(1550611448, 0),
-		Mode:  0755 | os.ModeDir,
-		Size:  96,
-	}, attr)
+	expectedAttr := plugin.EntryAttributes{}
+	expectedAttr.SetAtime(time.Unix(1550611510, 0))
```

Comment:
This is a bit verbose. I was thinking of using the builder pattern, but it's kind of weird in Go. Something like
```
expectedAttr.
  SetAtime(<time>).
  SetCtime(<time>).
  SetMtime(<time>)
```

Dunno if that reads well. I don't personally mind it.

Reply:
I don't mind it either.